### ISSUE TITLE
Vaadin 14 Grid Item double clicking.

### ISSUE THREAD

**Issue by rodolforfq** [mvysny/karibu-testing#54]
Hello there. First of all, thanks for this great library 😄 

I am currently testing a project and everything has been going great so far, but just hit a dead end.

I can use GridKt._clickItem, but there's no way to trigger a double click on a grid's item, as allowed per grid.addItemDoubleClickListener() method.

Any ideas?

**Comment by rodolforfq** [comment]
Found this:
https://github.com/mvysny/karibu-testing/blob/master/karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Grid.kt#L519

Apparently using the same technique while changing the event type to ItemDoubleClickEvent<T> could do the trick.

I would try it out but I haven't learned/used Kotlin yet 😢. What do you think @mvysny ?

**Comment by rodolforfq** [comment]
Hello again 😅 
I managed to get it working with the following java code, following your technique.

```java
 public static <T> void doubleClickGridItem(Grid<T> grid, Integer rowIndex) {
     var key = grid.getDataCommunicator().getKeyMapper().key(GridKt._get(grid, rowIndex));
     var doubleClickEvent = new ItemDoubleClickEvent<>(grid, true, key, -1, -1, -1, -1, 1, 1, false, false, false, false);
     _fireEvent(grid, doubleClickEvent);
 }
```
Guess this could be closed, or marked as a feature request. Cheers!

**Comment by mvysny** [comment]
Hello, excellent job figuring the code out :+1: Exactly, it's "just" the matter of firing the event. With your permission I'll reuse your  code :+1:

**Comment by mvysny** [comment]
Fix will be present in Karibu-Testing 1.2.6
